src/samples/conference/public/scripts/index.js

```diff
@@ -48,38 +48,59 @@ const runSocketIOSample = function() {
     var subscribeForward = getParameterByName('forward') === 'true'?true:false;
     var isSelf = getParameterByName('self') === 'false'?false:true;
     conference = new Owt.Conference.ConferenceClient();
-    function renderVideo(stream){
-        let subscirptionForward=null;
-        function subscribeDifferentResolutionForward(forward, resolution){
-            subscirptionForward && subscirptionForward.stop();
-            subscirptionForward = null;
+    function createResolutionButtons(stream, subscribeResolutionCallback) {
+        let $p = $(`#${stream.id}resolutions`);
+        if ($p.length === 0) {
+            $p = $(`<div id=${stream.id}resolutions> </div>`);
+            $p.appendTo($('body'));
+        }
+        // Resolutions from settings.
+        for (const videoSetting of stream.settings.video) {
+            const resolution = videoSetting.resolution;
+            if (resolution) {
+                const button = $('<button/>', {
+                    text: resolution.width + 'x' +
+                        resolution.height,
+                    click: () => {
+                        subscribeResolutionCallback(stream, resolution);
+                    }
+                });
+                button.prependTo($p);
+            }
+        }
+        // Resolutions from extraCapabilities.
+        for (const resolution of stream.extraCapabilities.video.resolutions) {
+            const button = $('<button/>', {
+                text: resolution.width + 'x' +
+                    resolution.height,
+                click: () => {
+                    subscribeResolutionCallback(stream, resolution);
+                }
+            });
+            button.prependTo($p);
+        };
+        return $p;
+    }
+    function subscribeAndRenderVideo(stream){
+        let subscirptionLocal=null;
```

Comment:
What does `subscriptionLocal` mean?

Reply:
Previously, it was named "subscriptionForward". Cause I use this function both for mixed stream and forward stream, I delete "Forward" and add a word, "local variable" maybe.